uhabits-android/src/main/java/org/isoron/uhabits/activities/common/dialogs/NumberDialog.kt

```diff
@@ -36,16 +36,17 @@ class NumberDialog : AppCompatDialogFragment() {
         val appComponent = (requireActivity().application as HabitsApplication).component
         val prefs = appComponent.preferences
         view = CheckmarkPopupBinding.inflate(LayoutInflater.from(context))
-        arrayOf(view.yesBtn, view.skipBtn).forEach {
+        arrayOf(view.yesBtn).forEach {
```

Comment:
We don't have a yes button for number dialog right? Do we need this line at all?